src/Serilog.Sinks.AzureTableStorage/Sinks/AzureTableStorageWithProperties/AzureTableStorageWithPropertiesSink.cs

```diff
@@ -67,7 +64,10 @@ public AzureTableStorageWithPropertiesSink(CloudStorageAccount storageAccount, I
 		/// <param name="logEvent">The log event to write.</param>
 		public void Emit(LogEvent logEvent)
 		{
-			_table.Execute(TableOperation.Insert(AzureTableStorageEntityFactory.CreateEntityWithProperties(logEvent, _formatProvider, _additionalRowKeyPostfix)));
+		    var op = TableOperation.Insert(
+		        AzureTableStorageEntityFactory.CreateEntityWithProperties(logEvent, _formatProvider, _additionalRowKeyPostfix));
+            
+            _table.ExecuteAsync(op).Wait(_waitTimeoutMilliseconds);
```

Comment:
Wow, the repetition in this codebase is pretty heavy :-) Guess it's not one to fix this time around, but some refactoring seems highly desirable sometime in the future...